Extension/package.json

```diff
@@ -93,6 +117,454 @@
           "description": "%c_cpp.configuration.clang_format_style.description%",
           "scope": "machine-overridable"
         },
+        "C_Cpp.test_root": {
+          "type": "object",
+          "default": {},
+          "description": "test1",
+          "scope": "resource",
+          "properties": {
+            "test_property1": {
+              "type": "string",
+              "default": "value"
+            },
+            "test_property2": {
+              "type": "boolean",
+              "default": false
+            },
+            "test_property3": {
+              "type": "string",
+              "enum": [
+                "Enabled",
+                "Disabled"
+              ],
+              "default": "Enabled"
+            }
+          }
+        },
+        "C_Cpp.formatting.engine": {
```

Comment:
I thought we were going to keep C_Cpp.formatting a bit longer and you would just add the new enum values to it.